app/mailers/candidate_mailer.rb

```diff
@@ -1,6 +1,16 @@
 class CandidateMailer < ApplicationMailer
   layout 'candidate_email_with_support_footer'
 
+  def confirm_sign_in(candidate_user, device:)
```

Comment:
i think this can just be `candidate` wdyt?

Reply:
Yeah definitely can be, makes sense to be consistent with the code base 👍

Reply:
Should also probably extract the subject lines into the yml file